Runtime/Scripts/msCoreAPI.cs

```diff
@@ -459,7 +459,8 @@ internal struct TextureData {
 
 #region Material
 
-internal struct MaterialPropertyData {
+internal struct MaterialPropertyData : IMaterialPropertyData
```

Comment:
Ideally, the PR to refactor MaterialPropertyData should be separated from this PR to change the occlusion strength, but since the latter is very small, we can probably make an exception.
If we do combine them, could you please mention what is the reason for the refactoring as the bulk of the PR is for refactoring.

Reply:
sounds good, I changed the title as it goes into the history